build/Microsoft.DotNet.Cli.Test.targets

```diff
@@ -3,6 +3,7 @@
   <Import Project="test/ProjectsToTest.props" />
   <Import Project="test/TestPackageProjects.targets" />
   <Import Project="test/TestAssetProjects.targets" />
+  <Import Project="test/TestProjects.targets" />
```

Comment:
It seems like ProjectsToTest.props should be deleted, if I'm not mistaken.

---
In reply to: [70897573](https://github.com/dotnet/cli/pull/3859#discussion_r70897573) [](ancestors = 70897573)